core/invoker/src/main/scala/whisk/core/containerpool/ContainerProxy.scala

```diff
@@ -146,9 +146,9 @@ class ContainerProxy(
             // the container did not come up cleanly, so disambiguate the failure mode and then cleanup
             // the failure is either the system fault, or for docker actions, the application/developer fault
             val response = t match {
-              case WhiskContainerStartupError(msg) => ActivationResponse.whiskError(msg)
-              case BlackboxStartupError(msg)       => ActivationResponse.applicationError(msg)
-              case _                               => ActivationResponse.whiskError(t.getMessage)
+              case WhiskContainerStartupError(_) => ActivationResponse.whiskError(Messages.resourceProvisionError)
+              case BlackboxStartupError(_)       => ActivationResponse.applicationError(Messages.resourceProvisionError)
```

Comment:
@dubeejw this comes from the change here:

https://github.com/apache/incubator-openwhisk/pull/3159#issuecomment-356098093

Reply:
might be worth keeping the old message, more informative if it's truly a `pull` issue.

Reply:
I agree, I'll update the inner messages accordingly.